components/Collections/src/MDCCollectionViewController.m

```diff
@@ -248,9 +269,10 @@ - (UIEdgeInsets)insetsAtSectionIndex:(NSInteger)section {
   return insets;
 }
 
-- (CGSize)inlaidSizeAtIndexPath:(NSIndexPath *)indexPath withSize:(CGSize)size {
+- (CGSize)inlaidSizeAtIndexPath:(NSIndexPath *)indexPath
+                       withSize:(CGSize)size
+                 collectionView:(UICollectionView *)collectionView {
   // If object is inlaid, return its adjusted size.
-  UICollectionView *collectionView = self.collectionView;
   if ([_styler isItemInlaidAtIndexPath:indexPath]) {
     CGFloat inset = MDCCollectionViewCellStyleCardSectionInset;
     UIEdgeInsets inlayInsets = UIEdgeInsetsZero;
```

Comment:
Should `_collectionViewLayout` below be the one attached to `collectionView` or the one this controller holds?